microsoft-edge/webview2/how-to/chromium-devtools-protocol.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Use the Chrome DevTools Protocol in WebView2 apps
+title: Use the Chrome DevTools Protocol (CDP) in WebView2 apps
 description: How to use the Chrome DevTools Protocol in your WebView2 app by using the Microsoft Edge WebView2 Chrome DevTools Protocol NuGet package.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 05/06/2021
+ms.date: 04/27/2023
 ---
-# Use the Chrome DevTools Protocol in WebView2 apps
+# Use the Chrome DevTools Protocol (CDP) in WebView2 apps
 
 The [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol) provides APIs to instrument, inspect, debug, and profile Chromium-based browsers.  The Chrome DevTools Protocol is the foundation for the Microsoft Edge DevTools.  Use the Chrome DevTools Protocol for features that aren't implemented in the WebView2 platform.
 
 To use the Chrome DevTools Protocol API in a WebView2 app, do either of the following:
 
-*  Install and use the [Microsoft.Web.WebView2.DevToolsProtocolExtension (Preview)](https://www.nuget.org/packages/Microsoft.Web.WebView2.DevToolsProtocolExtension) NuGet package (.NET).
```

Comment:
@novac42 @nishitha-burman -- CDP isn't in Technical Preview anymore, is it?  Removed the 5 hits on "preview" in this .md file.

Reply:
https://www.nuget.org/packages/Microsoft.Web.WebView2.DevToolsProtocolExtension